tests/MongoDB.Bson.Tests/Jira/CSharp783Tests.cs

```diff
@@ -23,6 +23,7 @@
 
 namespace MongoDB.Bson.Tests.Jira.CSharp783
 {
+#if WINDOWS
```

Comment:
I see that the preprocessor symbol `WINDOWS` *is* defined now (I don't believe it was before).

But the way it is defined seems fragile. I see that it is defined only in some `.csproj` files (presumably only the ones where it is needed). 

If we should happen to forget to define the `WINDOWS` preprocessor symbol this code will be silently commented out and we would be none the wiser.

Maybe this is the best we can do..., but it worries me.

I think we should have a JIRA ticket to **remove** all occurrences of `#if WINDOWS` in the future.

Reply:
Is there an existing preprocessor symbol we can use instead?

Maybe `_WIN64`?

Reply:
Also... from a previous comment I made that is no longer visible in the files view:

Skipping tests is good in the sense that they get reported as skipped. If you #ifdef them away it's as if they never even existed.

But if you do decide to skip tests (instead of #ifdefing them away) we should follow the existing pattern for other test requiremenst and do something like:

    RequirePlatform.Check().Platform(...)

Reply:
If you used `RequirePlatform` instead of `#if WINDOWS` we might not need the `WINDOWS` preprocessor symbol at all...

Reply:
I think we can use the solution from here: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/163/files#diff-c05284cb429957908b8ffcd50b29c61eb26c0af4871e58699080d527b0ec2ffbR37

Reply:
I can work on this change in the score of my ticket

Reply:
>
>Is there an existing preprocessor symbol we can use instead?
>Maybe _WIN64?

I never heard/seen this and wasn't able to google it now